.env.template

```diff
@@ -1,9 +1,44 @@
-ALLOWED_EMAIL_DOMAINS=
-SECRET_KEY=
-JWT_ACCESS_TOKEN_EXPIRES_MINUTES=
-SESSION_TYPE=
-CLIENT_PUBLIC_KEYS_DIRECTORY=
-SERVER_PUBLIC_KEY=
-SERVER_PRIVATE_KEY=
-SESSION_LIFETIME_MINUTES=
-REDIRECT_URL_TO_LOUIS_FRONTEND=
+# Server settings
```

Comment:
split off environment variables that must be setup vs environment variables that have reasonable values (commented out, with default value shown)

Reply:
not done

Reply:
done